src/components/application_manager/test/mobile_message_handler_test.cc

```diff
@@ -128,8 +128,8 @@ class MobileMessageHandlerTest : public testing::Test {
 
     // Act
     size_t payload_size = data.size();
-    Message* message = HandleIncomingMessage(
-        protocol_version, json_plus_binary_data, payload_size);
+    const auto message = std::shared_ptr<Message>(HandleIncomingMessage(
```

Comment:
@SNiukalov Can't see any sharing here. What is teh reason to create shared_ptr?

Reply:
@AByzhynar 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/41/commits/be4a738c2392d04c4fd0cd9a2dee39ce76b94c6e